src/lib/stitching/gravity/v2/stitching.ts

```diff
@@ -1204,3 +1244,13 @@ export const gravityStitchingEnvironment = (
     },
   }
 }
+
+export const formatSearchCriteriaAmount = (amountInCents: number, args) => {
+  const formattedAmount = amount((_) => amountInCents).resolve(
+    {
+      currencyCode: "usd",
+    },
+    args
+  )
```

Comment:
Here we pass the cents int our money helper and return the value. Note how we're defaulting to USD!

Reply:
++